test-apps/display-test-app/src/frontend/DisplayTestApp.ts

```diff
@@ -179,9 +181,22 @@ const dtaFrontendMain = async () => {
     const iModelName = configuration.iModelName;
     if (undefined !== iModelName) {
       const writable = configuration.openReadWrite ?? false;
-      iModel = await openFile(iModelName, writable);
+      iModel = await openFile({ fileName: iModelName, writable });
       setTitle(iModel);
     }
+    const origStandalone = configuration.standalone;
+    try {
+      const iModelId = configuration.iModelId;
+      const iTwinId = configuration.iTwinId;
+      if (undefined !== iModelId && undefined !== iTwinId) {
```

Comment:
Weird that if both iModelName and iTwin/ModelId are defined we potentially (try to) open two different iModels?

Reply:
Good point. Fixed.